cloud/pkg/common/messagelayer/context_test.go

```diff
@@ -18,78 +18,205 @@ package messagelayer
 
 import (
 	"reflect"
+	"sync"
 	"testing"
 
 	"github.com/kubeedge/beehive/pkg/common"
 	beehiveContext "github.com/kubeedge/beehive/pkg/core/context"
 	"github.com/kubeedge/beehive/pkg/core/model"
+	"github.com/kubeedge/kubeedge/cloud/pkg/common/modules"
 )
 
 const (
 	sendModuleName     = "testcore"
 	receiveModuleName  = "testcore"
 	responseModuleName = "testcore"
+	routerModuleName   = "router"
```

Comment:
any specific intention here? instead, can we just align with the other names here for test purpose?

```suggestion
	routerModuleName   = "testrouter"
```